frontend/test/metabase/lib/formatting.unit.spec.js

```diff
@@ -1,4 +1,5 @@
-import { isElementOfType } from "react-dom/test-utils";
+import { render, unmountComponentAtNode } from "react-dom";
+import { act, isElementOfType } from "react-dom/test-utils";
```

Comment:
Can we please use `@testing-library/react` here? It should also let us avoid wrapping every render in `act`